lib/rofl_ofdpa_fm_driver.cpp

```diff
@@ -694,7 +694,7 @@ cofflowmod rofl_ofdpa_fm_driver::enable_policy_arp(uint8_t ofp_version,
   return fm;
 }
 
-cofflowmod rofl_ofdpa_fm_driver::enable_policy_lldp(uint8_t ofp_version) {
+cofflowmod rofl_ofdpa_fm_driver::enable_policy(uint8_t ofp_version, const rofl::caddress_ll &mac, const uint16_t type) {
```

Comment:
Can we get a more expressive function name here

Reply:
sure, what do you want?